packages-presets/preset-rem-to-px/package.json

```diff
@@ -38,6 +38,7 @@
     "test:attw": "attw --pack --config-path ../../.attw-esm-only.json"
   },
   "dependencies": {
-    "@unocss/core": "workspace:*"
+    "@unocss/core": "workspace:*",
+    "@unocss/preset-wind4": "workspace:*"
```

Comment:
I think it doesn't need to install additional dependency.

Reply:
if remove it，will build fail

Reply:
I mean, keep `preset-rem-to-px` the original unchanged and make it a pure package.
I don't think it has anything relationship with `preset-wind4`.

Reply:
Get